traveler_core_kit/src/androidTest/java/com/guestlogix/travelercorekit/TravelerTest.java

```diff
@@ -129,8 +129,8 @@ public void onWishlistFetchSuccess(WishlistResult result, int identifier) {
                 String targetProductId = productIdReference.reference;
                 Product targetProduct = null;
                 for (CatalogItem item : result.getItems()) {
-                    if (item.getId().equals(targetProductId)) {
-                        targetProduct = item;
+                    if (item instanceof BookingItem && ((BookingItem) item).getId().equals(targetProductId)) {
```

Comment:
it might not be related to this change but are we using the real api to integrate test ?

Reply:
yes, using real api - usually rc server